src/lib/y2packager/clients/inst_productsources.rb

```diff
@@ -354,7 +354,8 @@ def NormalizeURL(url_string)
       return url_string if url_string.nil? || url_string == ""
 
       # unescape it
-      url_string = URI.unescape(url_string)
+      url_string = CGI.unescape(url_string)
+      log.info "unescaped url_string #{url_string.inspect}"
```

Comment:
URLs might potentially contain sensitive data like password, URLs should be always filtered using the `Yast::URL.HidePassword(url)` call when logging them.